mods/mc_teacher/gui_dash.lua

```diff
@@ -206,9 +213,11 @@ local function get_player_list_formspec(player, context)
     do
         fs[#fs + 1] = "tabheader[0,0;6.7,0.8;group;"
         fs[#fs + 1] = FS "All"
+
         local selected_group_idx = 1
         local i = 2
-        for name, group in pairs(minetest_classroom.get_all_groups()) do
+
+        for name, group in pairs(minetest_classroom.get_all_groups(player)) do
```

Comment:
`minetest_classroom.get_all_groups()` as defined on [line 77 of `api.lua`](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/109/files?diff=unified&w=1#diff-b84c94f3b3843d24b408d0d2938cfce2cefbf9892cd5cfbd3db3e2c8b04e5c6cR77) does not take any arguments

Reply:
Great catch; I'm not sure why I changed this line to add the player param. Will revert this change asap.